sdk/internal/recording/recordings/TestStartStop.json

```
@@ -51,10 +51,10 @@
         "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
         "var wlc=function(n,t,i){var u,f,r;n\u0026\u0026Identity\u0026\u0026(u=Identity.popupLoginUrls)\u0026\u0026(f=u.WindowsLiveId)\u0026\u0026Identity.wlProfile\u0026\u0026(r=_d.createElement(\u0022iframe\u0022),r.style.display=\u0022none\u0022,r.src=f\u002B\u0022\u0026checkda=1\u0022,r.setAttribute(\u0022data-priority\u0022,\u00222\u0022),_d.body.appendChild(r),i\u0026\u0026t\u0026\u0026t(\u0022SRCHHPGUSR\u0022,\u0022WTS\u0022,i,1,\u0022/\u0022))};\n",
         "//]]\u003E\u003C/script\u003E\u003Cscript type=\u0022text/rms\u0022\u003E//\u003C![CDATA[\n",
-        "(function() { var conditionalSignInParams ={\u0022notifEnabled\u0022:false,\u0022notifFetchAsync\u0022:false}; BingAtWork.ConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn(conditionalSignInParams); })();;(function() { var config ={\u0022url\u0022:\u0022https%3a%2f%2flogin.microsoftonline.com%2fcommon%2foauth2%2fauthorize%3fclient_id%3d9ea1ad79-fdb6-4f9a-8bc3-2b70f96e34c7%26response_type%3did_token%2bcode%26nonce%3de7627bed-8863-43dd-b8c6-b71eab6a1684%26redirect_uri%3dhttps%253a%252f%252fwww.bing.com%252forgid%252fidtoken%252fconditional%26scope%3dopenid%26response_mode%3dform_post%26msafed%3d0%26prompt%3dnone%26state%3d%257b%2522ig%2522%253a%2522DB5D90E3764548E492F0279FC0C5B536%2522%257d\u0022,\u0022sandbox\u0022:\u0022allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts\u0022,\u0022currentEpoch\u0022:\u00221629906926000\u0022}; sj_evt.fire(\u0027ssoFrameExists\u0027, config); })();;\n",
+        "(function() { var conditionalSignInParams ={\u0022notifEnabled\u0022:false,\u0022notifFetchAsync\u0022:false}; BingAtWork.ConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn \u0026\u0026 BingAtWork.ConditionalSignIn.bindToConditionalSignIn(conditionalSignInParams); })();;(function() { var config ={\u0022url\u0022:\u0022https%3a%2f%2flogin.microsoftonline.com%2fcommon%2foauth2%2fauthorize%3fclient_id%3d9ea1ad79-fdb6-4f9a-8bc3-2b70f96e34c7%26response_type%3did_token%2bcode%26nonce%3d1dd4b886-0fe4-4f40-9648-29978ab40465%26redirect_uri%3dhttps%253a%252f%252fwww.bing.com%252forgid%252fidtoken%252fconditional%26scope%3dopenid%26response_mode%3dform_post%26msafed%3d0%26prompt%3dnone%26state%3d%257b%2522ig%2522%253a%2522972CE6E173FF43899851DA15BE017233%2522%257d\u0022,\u0022sandbox\u0022:\u0022allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts\u0022,\u0022currentEpoch\u0022:\u00221631561489000\u0022}; sj_evt.fire(\u0027ssoFrameExists\u0027, config); })();;\n",
         "//]]\u003E\u003C/script\u003E\u003C/div\u003E\u003C/body\u003E\u003C/html\u003E"
       ]
     }
   ],
   "Variables": {}
-}
+}
```

The recorder should be updated to ensure it adds the EOF newline
```suggestion
}

```

---

That addition is en route and hopefully will be added in the next update.